api/openapi.gen.yaml

```diff
@@ -558,6 +604,59 @@ components:
         source_id:
           type: string
       type: object
+    v1.GCPReservationRequest:
+      properties:
+        amount:
+          format: int64
+          type: integer
+        image_id:
+          type: string
+        machine_type:
+          type: string
+        poweroff:
+          type: boolean
+        pubkey_id:
+          format: int64
+          type: integer
+        source_id:
+          type: string
+        zone:
+          type: string
+      type: object
+    v1.GCPReservationResponse:
+      properties:
+        amount:
+          format: int64
+          type: integer
+        gcp_operation_name:
+          type: string
+        image_id:
+          type: string
+        instances:
+          items:
+            properties:
+              instance_id:
+                type: string
+              reservation_id:
+                format: int64
+                type: integer
```

Comment:
This field is exposed for no reason tho, I mean this is reservation response, there is no need to copy reservation_id N times, it has zero information since the user performed GET call with reservation ID already.

Now since any refactoring would cause pains, what I suggest is to merge this as is and ask @amirfefer to hide this field (by simply copying only relevant fields into the payload type from model type).